src/python/director/startup.py

```diff
@@ -1,6 +1,8 @@
 # This script is executed in the main console namespace so
 # that all the variables defined here become console variables.
 
+from __future__ import division
```

Comment:
im a wimp and so i am concerned about adding this to startup.py.  I don't mind using it in modules, but this effects everything run in the python console.  if you think it should be here then maybe i can be convinced.